server/request_handler.go

```diff
@@ -101,6 +102,13 @@ func (r *RpcRequestHandler) process() {
 	// Parse JSON RPC payload
 	var jsonReq *types.JsonRpcRequest
 	if err = json.Unmarshal(body, &jsonReq); err != nil {
+		r.requestRecord.UpdateRequestEntry(r.req, http.StatusBadRequest, err.Error())
+		r.logger.Warn("[process] Parse payload", "error", err)
+		(*r.respw).WriteHeader(http.StatusBadRequest)
+		return
+
+		// Deprecate handling batch - DO NOT PROCEED
+		// Note: go vet checks are disabled for unreachable code
```

Comment:
you could keep the linter and comment the code. but not important i think :)

Reply:
There are lot of code need to be commented.Linter does't allow. better to suppress linter warnings